openshift/mlflow-experiment-job-template.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+
+labels:
+  application: ${APPLICATION_NAME}
+
+metadata:
+  name: mlflow-experiment-job-template
+
+parameters:
+- description: Application name
+  value: 'mlflow-experiment'
+  name: APPLICATION_NAME
+  required: true
+- name: MLFLOW_TRACKING_URI
+  value: http://mlflow-server-test-route-aiops-kubeflow.cloud.paas.upshift.redhat.com/
+  required: false
+- name: APP_IMAGE_NAME
+  description: Name of the application image to be pulled from the registry
+  value: mlflow-experiment-image
+  required: true
+
+
+objects:
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: ${APPLICATION_NAME}
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    template:
+      metadata:
+        name: ${APPLICATION_NAME}
+      spec:
+        containers:
+        - name: ${APPLICATION_NAME}
+          image: ${APP_IMAGE_NAME}:latest
+          imagePullPolicy: Always
+          resources:
+            requests:
+              memory: 500Mi
+            limits:
+              memory: 1Gi
+          env:
+          - name: PARAM_ALPHA     # Using env variables to pass parameters
+            value: "0.1"
```

Comment:
these `PARAMS` must not be in the template. They are being set when creating a job from the template

Reply:
+1 We should remove the PARAM_ALPHA, PARAM_L1_RATION as this is a custom parameter passed in to the mlflow sample code.

Reply:
They were supposed to be examples, but since DSs are not going to use this template, I will remove them